### DISCUSSION TITLE
why

### DISCUSSION THREAD

**Discussion by ngewa** [itzg/docker-minecraft-bedrock-server#590] [Q&A]
![Screenshot_20240901-103555_Termux](https://github.com/user-attachments/assets/8fffd172-77f3-4afb-a611-486071e565ef)
error 
https://github.com/itzg/docker-minecraft-bedrock-server 

-bash: ./bedrock_server: cannot execute binary file: Exec format error

**Comment by itzg** [comment]
Please provide your container configuration.

**Comment by ngewa** [comment]
NAME="Ubuntu"
VERSION="20.04.6 LTS (Focal Fossa)"                                                     ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.6 LTS"                                                        VERSION_ID="20.04"

**Comment by ngewa** [comment]
OS: Ubuntu 20.04.6 LTS aarch64
Host: KVM Virtual Machine virt-4.2
Kernel: 5.4.0-193-generic

**Comment by itzg** [comment]
So you're not running the docker image? Do note that Bedrock Dedicated Server is Mojang/Microsoft's software -- my image just automates the download and setup. 

They only provide x86 binaries so you can't run on aarch64/ARM without an emulator. If you were using the docker image it would take care of that.

**Comment by ngewa** [comment]
but if you use docker image

want to set
server.properties where is it in the path if checked using pwd
